source/quickstart.rst

```diff
@@ -0,0 +1,477 @@
+
+  >>> rks_h2o.xc = '.2 * HF + .08 * LDA + .72 * B88, .81 * LYP + .19 * VWN' # B3LYP
+  >>> rks_h2o.grids.atom_grid = (100, 770)
+  >>> rks_h2o.grids.prune = None
+  >>> e_rks = rks_h2o.kernel()
+  
+The use of a combination of dense and sparse grids are particularly important whenever XC functionals with non-local dispersion (van der Waals) corrections are employed (cf. `dft/33-nlc_functionals.py <https://github.com/pyscf/pyscf/blob/master/examples/dft/33-nlc_functionals.py>`_):
+
+  >>> rks_c2 = dft.RKS(mol_c2)
+  >>> rks_c2.xc = 'wb97m_v'
+  >>> rks_c2.nlc = 'vv10'
+  >>> rks_c2.grids.atom_grid = (99,590)
+  >>> rks_c2.grids.prune = None
+  >>> rks_c2.nlcgrids.atom_grid = (50,194)
+  >>> rks_c2.nlcgrids.prune = dft.gen_grid.sg1_prune
+
+.. _TDMF:
+
+Time-Dependent Mean-Field Theory
+--------------------------------
+
+Linear response theory has been implemented for both HF and KS-DFT (cf. `tddft/00-simple_tddft.py <https://github.com/pyscf/pyscf/blob/master/examples/tddft/00-simple_tddft.py>`_):
+
+  >>> from pyscf import tdscf
+  >>> tdhf_h2o = tdscf.TDHF(rhf_h2o)
+  >>> tdhf_h2o.nstates = 6
+  >>> tdhf_h2o.kernel()
+  >>> tddft_h2o = tdscf.TDA(rks_h2o) # TDDFT with Tamm-Dankoff approximation
+  >>> tddft_h2o.nstates = 4
+  >>> tddft_h2o.kernel()
+
+From a converged time-dependent mean-field calculation, the corresponding natural transition orbitals for a particular excited state may be recovered as (cf. `tddft/01-nto_analysis.py <https://github.com/pyscf/pyscf/blob/master/examples/tddft/01-nto_analysis.py>`_):
+
+  >>> weights, nto = tdhf_h2o.get_nto(state=2)
+  
+As an alternative to response theory, :math:`\Delta`-SCF with Gill's maximium occupation method has been implemented for calculating specific excited states, cf. `scf/50-mom-deltaSCF.py <https://github.com/pyscf/pyscf/blob/master/examples/scf/50-mom-deltaSCF.py>`_.
+
+.. _LOC:
+
+Spatially Localized Molecular Orbitals
+--------------------------------------
+
+PySCF offers a number of different standard schemes for localizing MOs, e.g., Pipek-Mezey, Foster-Boys, and Edmiston-Ruedenberg (cf. `local_orb/03-split_localization.py <https://github.com/pyscf/pyscf/blob/master/examples/local_orb/03-split_localization.py>`_):
+
+  >>> from pyscf import lo
+  >>> occ_orbs = rhf_h2o.mo_coeff[:, rhf_h2o.mo_occ > 0.]
+  >>> fb_h2o = lo.Boys(mol_h2o, occ_orbs, rhf_h2o) # Foster-Boys
+  >>> loc_occ_orbs = fb.kernel()
+  >>> virt_orbs = rhf_h2o.mo_coeff[:, rhf_h2o.mo_occ == 0.]
+  >>> pm_h2o = lo.Boys(mol_h2o, virt_orbs, rhf_h2o) # Pipek-Mezey
+  >>> loc_virt_orbs = pm.kernel()
+  
+In addition, Knizia's intrinsic bond orbitals have been implemented (cf. `local_orb/04-ibo_benzene_cubegen.py <https://github.com/pyscf/pyscf/blob/master/examples/local_orb/04-ibo_benzene_cubegen.py>`_):
```

Comment:
I think there's two implementations of IAO in PySCF

Reply:
Please elaborate?

Reply:
From https://github.com/pyscf/pyscf/issues/820: first alternative
```
lmo_coeff = ibo(mol, mf.mo_coeff[:, mf.mo_occ > 0], locmethod='IBO', exponent=2)
```
Second alternative
```
loc = PipekMezey(mol=mol, mf=mf)
loc.pop_method = 'iao'
lmo_coeff = loc.kernel(mf.mo_coeff[:, mf.mo_occ > 0])
```
The second one also supports several other charge models in accordance to our generalized Pipek-Mezey scheme.

Reply:
Sure, forgot about that. Do you want me to update the quickstart with that second example or should that rather go into the user guide?

Reply:
@susilehtola - would you like me to update the `IBO` example with the `PipekMezey` example above? And is this what's blocking the PR?